easysid.cpp

```diff
@@ -14,10 +14,53 @@
 #endif
 
 SID sid;
-PcmOutput *pcm;
+PcmOutput *pcm = NULL;
+PcmOutput *wav = NULL;
+int sample_rate = 44100;
 
 extern "C" {
 
+EXPORT void start_capture(const char *fn)
+{
+    if (wav != NULL) {
+        // Deleting the existing object will flush, and close the current capture file.
+        delete wav;
+        wav = NULL;
+    }
+    wav = makePcmOutputWav(fn, sample_rate);
+}
+
+EXPORT void end_capture()
+{
+    if (wav != NULL) {
+        wav->flush();
+        delete wav;
+        wav = NULL;
+    }
+}
+
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        // TODO: Implement errors, or SEH to handle these scenarios.
+        return;
+    }
+    sid.set_chip_model(static_cast<chip_model>(chip));
+}
+
+EXPORT void set_sample_parameters(double clock, uint32_t sample_mode, uint32_t sample_freq)
```

Comment:
This function has nothing to do with capture. It should be in a separate commit.